lib/workers/pr/changelog/release-notes.ts

```diff
@@ -4,42 +4,72 @@ import MarkdownIt from 'markdown-it';
 import * as URL from 'url';
 
 import { api } from '../../../platform/github/gh-got-wrapper';
+import { api as api_gitlab } from '../../../platform/gitlab/gl-got-wrapper';
 import { logger } from '../../../logger';
 import { ChangeLogResult, ChangeLogNotes } from './common';
 
 const { get: ghGot } = api;
+const { get: glGot } = api_gitlab;
 
 const markdown = new MarkdownIt('zero');
 markdown.enable(['heading', 'lheading']);
 
 export async function getReleaseList(
-  githubApiBaseURL: string,
+  apiBaseURL: string,
   repository: string
 ): Promise<ChangeLogNotes[]> {
   logger.trace('getReleaseList()');
   // istanbul ignore if
-  if (!githubApiBaseURL) {
+  if (!apiBaseURL) {
+    logger.debug('No apiBaseURL');
     return [];
   }
   try {
-    let url = githubApiBaseURL.replace(/\/?$/, '/');
-    url += `repos/${repository}/releases?per_page=100`;
-    const res = await ghGot<
-      {
-        html_url: string;
-        id: number;
-        tag_name: string;
-        name: string;
-        body: string;
-      }[]
-    >(url);
-    return res.body.map((release) => ({
-      url: release.html_url,
-      id: release.id,
-      tag: release.tag_name,
-      name: release.name,
-      body: release.body,
-    }));
+    let url = apiBaseURL.replace(/\/?$/, '/');
+    if (apiBaseURL.includes('github')) {
```

Comment:
Can this logic instead be moved to the `else` part of the gitlab if? I'm concerned that if someone names their GHE `ghe.company.com` then they won't get release notes, so ideal to make it more backwards compatible.

Reply:
I moved it outside of the if, as an else wasn't needed because there's a return inside (the linter caught this).